lib/experimental/Overlays/Dialog/index.tsx

```diff
@@ -1,37 +1,42 @@
 import { Button, ButtonProps } from "@/components/Actions/Button"
-import { Icon, IconType } from "@/components/Utilities/Icon"
+import {
+  AlertAvatar,
+  type Props as AlertAvatarProps,
```

Comment:
let's export better named props instead of generic `Props` from the `AlertAvatar`